yarn-project/txe/src/oracle/txe_oracle.ts

```diff
@@ -772,8 +772,12 @@ export class TXE implements TypedOracle {
 
     const executionResult = await this.executePublicFunction(targetContractAddress, args, callContext);
 
```

Comment:
There was a bug in TXE which resulted in a test passing even when an error was thrown. Grego fixed it in this PR and what follows are changes made by him.